great_expectations/profile/base.py

```diff
@@ -163,7 +163,7 @@ class ProfilerTypeMapping:
         "string",
         "str",
         "object",
-        "dtype('O')",
+        'dtype("O")',
```

Comment:
I had to change it to pass the tests because now we don't allow single quotes in json serialization

Reply:
I finally canceled this modification